### ISSUE TITLE
LP withdraw issues 

### ISSUE THREAD

**Issue by 0xean** [shapeshift/web#7708]
- vague rumors in discord of LP withdraws not working @gomesalexandre please add details

**Comment by 0xean** [comment]
https://viewblock.io/thorchain/tx/606AC20784E3A3812D6F83B3E67D1D419B4AD0758B9CD6BB142DC5E98CF080B5

failed tx

**Comment by gomesalexandre** [comment]
The failed Tx above is caused by the new `LIQUIDITYLOCKUPBLOCKS` (which we absolutely *should* handle in the UI, there was no notion of withdraw lockup time until recently).

Retested with older positions (sym and asset on both sides) and was able to withdraw:

- Asym asset:
  <img width="483" alt="image" src="https://github.com/user-attachments/assets/90634c5a-453d-4262-8ee1-db208a69eb2b">
  <img width="451" alt="image" src="https://github.com/user-attachments/assets/efacc00c-d497-4e75-89df-0547d8a7a44c">
- Asym RUNE: https://viewblock.io/thorchain/tx/FE3BE7549136777827D46F46049C7E26FA047128BB4E60D7D5D082EB1617B84B
  Note Tx is failed and not even observed, viewblock/runescan is hitting the mempool. That's probably the case I was seeing and unrelated to the hardfork / liquidity lockup shenanigans
- Sym: TBD in one hour because of the lockup period

@twblack88 @0xean this is nothing new, we just happened to have noticed it after the hardfork.

I've added the `needs-product` and `needs-grooming` tags, and moved this back to the backlog so we groom this tonight - we will need to accommodate for the new lockup period time (currently defined as 600 blocks in mimir, or 3600 seconds/ 1 hour).

Also created https://github.com/shapeshift/web/issues/7710 to guard against amount too lows for withdraw
